website_compassion/template/my_account_donations.xml

```diff
@@ -218,8 +218,9 @@
                     <t t-set="children" t-value="invoice.invoice_line_ids.mapped('contract_id.child_id')"/>
                     <div class="row">
                         <div class="col-6">
-                            <label><t t-esc="invoice.get_date('date_invoice', 'MMMM Y').title()"/></label>
+                            <label><t t-esc="invoice.get_date('date_invoice', 'MMMM yyyy').title()"/></label>
                         </div>
+                        <div class="col-3">
```

Comment:
adding this line is a typo, can you  make a new pr with just this removal (as this pr was merged)